deployment/terraform/create.go

```diff
@@ -149,6 +149,22 @@ func (t *Terraform) Create(initData bool) error {
 		}
 	}
 
+	if t.output.HasDB() {
+		// Aurora sets the default_text_search_config parameter to 'simple';
+		// a more sane default for a generic deployment is 'english'
+		if t.config.TerraformDBSettings.InstanceEngine == "aurora-postgresql" {
+			if err := t.setDefaultTextSearchConfig(extAgent, "pg_catalog.english"); err != nil {
+				return fmt.Errorf("could not modify default_search_text_config: %w", err)
+			}
+		}
+
+		if initData {
```

Comment:
@agarciamontoro - Would this be restricted based on if you have a DB or not? The way I'm reading this is that it's just if you're having it initialize data, which wouldn't be 100% tied to whether it's making a new DB or not, would it?

Reply:
Yes, there is no need to move this condition. This should remain under `output.HasAppServers()`

Reply:
It could be argued that the DB should be modified only in the case it's deployed by Terraform, and assume that the user wants full control (so no changes in the config nor in the initial data) if the DB is already provided by them.
Thoughts, @agnivade?

Reply:
Oh yeah good point. Approved.